src/groovy/com/unifina/utils/ThreadUtil.groovy

```diff
@@ -0,0 +1,19 @@
+package com.unifina.utils
+
+class ThreadUtil {
```

Comment:
Why is this class useful? I'm not sure I understand why it was added, so maybe a comment would be useful.